assets/js/modules/ads/components/settings/SettingsForm.js

```diff
@@ -48,6 +49,15 @@ export default function SettingsForm( { hasModuleAccess } ) {
 		<div className="googlesitekit-ads-settings-fields">
 			<StoreErrorNotices moduleSlug="ads" storeName={ MODULES_ADS } />
 
+			<div className="googlesitekit-setup-module__inputs">
+				<AdsConversionIDTextField
+					helperText={ __(
+						'The Ads conversion ID will help track the performance of ad campaigns for the corresponding account',
```

Comment:
The helper text is also different in Figma, again, have we made a deliberate decision to use this copy instead?

![image](https://github.com/google/site-kit-wp/assets/18395600/ae3d3e49-391c-4965-9456-7ba9138aff8b)